boards/arm/stm32h747i_disco/stm32h747i_disco_m7.dts

```diff
@@ -48,6 +48,27 @@
 	};
 };
 
+&clk_hse {
+	hse-bypass;
+	clock-frequency = <DT_FREQ_M(8)>; /* STLink 8MHz clock */
```

Comment:
On H747 disco, hse is not from STlink. So no bypass and freq=25MHz

Reply:
Txs, fixed